pkg/cmd/kafka/topic/update/update.go

```diff
@@ -302,7 +304,7 @@ func runCmd(opts *options) error {
 	}
 
 	opts.Logger.Info(opts.localizer.MustLocalize("kafka.topic.update.log.info.topicUpdated", topicNameTmplPair, kafkaNameTmplPair))
-	return dump.Formatted(opts.IO.Out, opts.outputFormat, response)
+	return nil
```

Comment:
Running update command with output flag set as yml or json have no effect. Should we remove the output flag.